incubator/hoverfly/values.yaml

```diff
@@ -0,0 +1,21 @@
+# Default values for hoverfly.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+image:
+  repository: docker.io/spectolabs/hoverfly
+  tag: v1.0.0-rc.2
+service:
+  type: ClusterIP
+  externalAdminPort: 8888
+  adminPort: 8888
+  externalProxyPort: 8500
+  proxyPort: 8500
+healthcheckEndpoint: /api/health
+hoverflyFlags:
+resources:
```

Comment:
Can you please comment out the resources by default.

Reply:
Fixed